src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.wasm.cs

```diff
@@ -7,11 +7,14 @@
 using Uno.Foundation;
 using System.Linq;
 using Uno.UI.UI.Xaml.Documents;
+using Microsoft.Extensions.Logging;
 
 namespace Windows.UI.Xaml.Controls
 {
 	partial class TextBlock : FrameworkElement
 	{
+		private Dictionary<Size, Size> _measureCache = new Dictionary<Size, Size>();
```

Comment:
You should hint the dictionary size here...

Reply:
It's 16 by default, what do you think there should be ?